ExcelReaderAPI/Controllers/AuthController.cs

```diff
@@ -0,0 +1,93 @@
+﻿
+using System.IdentityModel.Tokens.Jwt;
+using System.Security.Claims;
+using System.Security.Cryptography;
+using ExcelReaderAPI.Models;
+using ExcelReaderAPI.Services;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.IdentityModel.Tokens;
+
+namespace ExcelReaderAPI.Controllers
+{
+    [Route("api/auth")]
+    [ApiController]
+    public class AuthController : ControllerBase
+    {
+
+        private readonly UserAuthService _authServices;
+        private readonly IConfiguration _configuration;
+
+        public AuthController(UserAuthService authServices, IConfiguration configuration)
+        {
+            _authServices = authServices;
+            _configuration = configuration;
+
+        }
+
+
+
+        [HttpPost("register-user")]
+        public async Task<IActionResult> RegisterUser(UserDto request)
```

Comment:
No need of async as there is not await here

Reply:
removed